### ISSUE TITLE
Good price increase by 10 times

### ISSUE THREAD

**Issue by settintotrieste** [settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE#79]
This allegedly increases the chance for rebels to appear. More testing is needed to ensure it works correctly and does not throw the events and decisions into chaos again.

**Comment by ARR8** [comment]
About the rebels - what makes you say that? How does the math on the rebels work?

**Comment by EmperorRAG** [comment]
If I remember correctly, at 7 miltancy, a pop will start to contribute some of its income to the movement it is part of. The funds that the movement receive increase its likelihood of revolting. In the mod at the moment, majority of the world's pops are not able to satisfy their needs and so are not able to contribute to the movements. Increasing the prices of goods would increase the amount of cash the pops have and so allow them to fund movements to revolt. A balancing of the economy could also help in fixing pop revolts in the mod although increasing the prices of goods will provide an immediate effect.

**Comment by settintotrieste** [comment]
Thanks EmperorRag for the information on the rebels system. Based on a test game, the increase in good prices does increase the appearance of rebels with some complications.

However the increased presence of rebels does at times interfere with events, mainly the Civil War Mechanic which allows for the rebel coup and the actual rebel coup to occur simultaneously. The fix I pursued is to increase the necessary militancy required for the autorising of rebels to 10 such that the Civil War System can have enough time to occur. The second possible fix is to increase the MTTH for the coup so the rebels have enough time to do thier thing. 

The second problem is that some countries get overthrown by rebels too readily: Jordan, Saudi, North Vietnam and Indonesia. The fix I applied is to reduce the militancy generation for those specific countries so that the rebels will only appear once in a while.

The last problem is that the good price increase can cause a stack overflow error however longer testing is required to check if this occurs. 

If you managed to play a game to the end date or at least 2046 do let me know if there are any game-breaking issues. Thanks
